cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -46,13 +46,18 @@ type CustomArgs struct {
 	SkipUnsafe bool
 }
 
-// This is the comment tag that carries parameters for conversion generation.
+// There are the comment tags that carry parameters for conversion generation.
 const tagName = "k8s:conversion-gen"
+const externalTypesTagName = "k8s:conversion-gen-external-types"
```

Comment:
@mehdy sorry i made a change after you last time reviewed it. I added the `externalTypesTagName` for user to specify where the external types are, instead of squeeze it with the existing `tagName`. This is more consistent with the way i updated the default-gen (https://github.com/kubernetes/gengo/pull/54).